addons/account/data/service_cron.xml

```diff
@@ -10,4 +10,15 @@
         <field name="code">model._run_reverses_entries()</field>
         <field name="state">code</field>
     </record>
+
+    <record id="ir_cron_reverse_entry" model="ir.cron">
+        <field name="name">Account; Post draft entries up to today</field>
+        <field name="interval_number">1</field>
+        <field name="interval_type">days</field>
+        <field name="numbercall">-1</field>
+        <field name="doall" eval="False"/>
+        <field name="model_id" ref="model_account_move"/>
+        <field name="code">model._run_post_draft()</field>
+        <field name="state">code</field>
+    </record>
```

Comment:
I was expecting nextcall to be set... (something like 2AM) to avoid having the cron run at installation time, everyday. 
 That'd mean sometimes finding draft moves with date=today in the morning and them being posted during the day, which would be the source of tickets I guess.

Please, fix that run time

Reply:
I put hour=2, but it is still in server time... Should I find something better?

Reply:
no it's fine with server time